config/config.go

```diff
@@ -151,7 +150,7 @@ func ReadConfig(log zerolog.Logger, config []byte, inputPrefix, outputPrefix str
 		Composites:  composites,
 	}, nil
 }
-func ReadConfigFile(configFile string, inputPrefix, outputPrefix string) (*ControlToolConfig, error) {
+func ReadConfigFile(configFile string, configDir string, inputPrefix, outputPrefix string) (*ControlToolConfig, error) {
 	if configFile == "" {
 		return nil, fmt.Errorf("a --config file is required to be specified")
```

Comment:
If we specify a configDir, do we need to specify a configFile.

Reply:
I don't think so, but I fear that just having config-dir would break stuff. I'm not sure how we deploy vault-ctrl-tool around, but I can see that we would have instance still using the `vault-config.yml` and not the directory

Reply:
yeah makes sense. Especially when you consider the top level configuration contained in it.